ApiTests/Tests/Pages/DynamicControlsPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ApiTests.Tests.UI;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using TestProject.Utils;
+
+namespace TestProject.Tests.Pages
+{
+    internal class DynamicControlsPage : BaseTest
+    {
+        private static readonly By enableBtn = By.XPath(string.Format(XpathPatterns.preciseTextXpath, "Enable"));
+        private static readonly By disableBtn = By.XPath(string.Format(XpathPatterns.preciseTextXpath, "Disable"));
+        private static readonly By inputField = By.XPath(string.Format(XpathPatterns.typeText));
+        readonly WebDriverWait wait = new WebDriverWait(Browser.GetDriver(), TimeSpan.FromSeconds(100));
```

Comment:
It'd be better to create a separate class for waits (for example, WaitUtils or smth similar), so that we could reuse those methods without making them a part of a page class